packages/jetpack-blocks/package.json

```diff
@@ -26,7 +26,9 @@
     "@wordpress/token-list": "^1.1.0",
     "@wordpress/url": "^2.3.3",
     "classnames": "^2.2.6",
+    "circular-dependency-plugin": "5.0.2",
```

Comment:
Can we remove this now?

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/31503/commits/da7d36d56f2862d38451315dcd79f2735b44227c